skyportal/handlers/api/instrument.py

```diff
@@ -17,6 +22,15 @@ def post(self):
             telescope_id, self.current_user, raise_if_none=True, mode="read"
         )
 
+        if "field_data" in data:
+            field_data = data.pop("field_data")
```

Comment:
Is it verified that these fields will be in data?

Reply:
Yes added on the other PR.